src/k8s-engine/lima.ts

```diff
@@ -1059,6 +1056,34 @@ ${ commands.join('\n') }
     }
   }
 
+  /**
+   * Get IPv4 address for specified interface.
+   */
+  protected async getInterfaceAddr(itf: string) {
+    const ipAddr = await this.limaWithCapture('shell', '--workdir=.', MACHINE_NAME, 'ip', 'addr', 'show', itf);
+    const match = ipAddr.match(' inet ([0-9.]+)');
+
+    return match ? match[1] : '';
+  }
+
+  /**
+   * Display dialog to explain that bridged networking is not available.
+   */
+  protected async noBridgedNetworkDialog(sharedIP: string) {
```

Comment:
I wanted it to be "in your face" as that is an unexpected change. Now I'm unsure. If the user explicitly configured bridged networking, then I would definitely want to see an error dialog, but since it is a default now, maybe it is distracting?

I'm not really sold on making all ports accessible from outside the host by default, but that is something @mattfarina  insisted on.